### ISSUE TITLE
Zenodo not reflecting latest GitHub Release

### ISSUE THREAD

**Issue by peterjc** [zenodo/zenodo#2243]
Browser & OS (see also https://www.whatismybrowser.com/): Chrome & macOS

## Describe the bug

New release on made GitHub, but 5 days later and it has not appeared on linked Zenodo repository.

## Steps to Reproduce
1. Go to https://github.com/peterjc/thapbi-pict/releases
2. Note the latest release is v0.10.2 https://github.com/peterjc/thapbi-pict/releases/tag/v0.10.2 dated 2021-11-05:
3. Go to https://doi.org/10.5281/zenodo.4529395
4. Note this redirects to the v0.10.1 which remains the latest release on Zenodo, dated 2021-07-28
5. As the repository owner, go to https://zenodo.org/account/settings/github/repository/peterjc/thapbi-pict
6. Notice the latest entry is v0.10.1 with no entry (failed or otherwise) for v0.10.2

## Expected behavior

Zenodo should import any new release from GitHub within a day or two, or if there is an error inform the repository owner. e.g. Via the GitHub settings page: https://zenodo.org/account/settings/github/repository/peterjc/thapbi-pict

## Additional context

This had been working as I expected prior to the few months interval with no new releases. The ``.zenodo.json`` configuration file has not been touched during this time:

https://github.com/peterjc/thapbi-pict/commits/master/.zenodo.json

My GitHub account is still linked according to https://zenodo.org/account/settings/linkedaccounts/

**Comment by peterjc** [comment]
I made another new release today: https://github.com/peterjc/thapbi-pict/releases/tag/v0.10.3

I could see that it was "processing" here, and quickly "published": https://zenodo.org/account/settings/github/repository/peterjc/thapbi-pict/

So things seem to be working _now_, but no sign of the missing v0.10.2 release from earlier this month.

**Comment by astrojuanlu** [comment]
Observing the same problem, just created https://github.com/poliastro/poliastro/releases/tag/v0.16.0 and it's not reflected on https://zenodo.org/account/settings/github/repository/poliastro/poliastro.

Is there a way to "refresh" the list of releases?

**Comment by astrojuanlu** [comment]
By the way, the webhook was sent and Zenodo responded with HTTP 202 ACCEPTED. I redelivered the webhook, Zenodo replied 202 again, and still no luck.

**Comment by peterjc** [comment]
How did you manually trigger the webhook?

(I suspect this is a load issue, and retrying later will work)

**Comment by astrojuanlu** [comment]
You need to go to https://github.com/peterjc/thapbi-pict/settings/hooks, locate the zenodo webhook, go to "recent deliveries", check the response (optional), and click "redeliver"

**Comment by ines-cruz** [comment]
Dear @peterjc could I kindly ask that you reach out to us via our support channel at info@zenodo.org so we can look into this issue further?

**Comment by peterjc** [comment]
Done - although the automatic reply does say:

> For all future support requests we highly recommend using the contact form at https://zenodo.org/support

**Comment by peterjc** [comment]
My missing release was fixed by Zenodo support, who explained that according to their logs Zenodo never received the notification from GitHub when I made that release. i.e. This appears to have been a GitHub outage.

Zenodo support's suggested workaround was to tweak the "missing" GitHub release (e.g. a trivial edit to the release text), which ought to re-notify Zenodo.

I will close this for now, but @astrojuanlu let me know if you'd rather we left it open to address your missing poliastro  https://doi.org/10.5281/zenodo.593610 release?
